OpenEdXMobile/res/values/colors.xml

```diff
@@ -77,5 +78,10 @@
     <color name="view_pager_indicator_color_inactive">#282f3749</color>
     <color name="custom_dark_gray">#083954</color>
     <color name="edx_input_border_color">#dbe0e9</color>
-    <color name="dark_blue">#083954</color>
+
+    <!-- Colors defined for PhilU App -->

```

Comment:
As per discussion in person, sooner or later we should use a common color palette for whole app theme.

Reply:
We will refactor color palette later